config.go

```diff
@@ -87,70 +87,69 @@ func (c Configs) validateMappingFile() error {
 	return nil
 }
 
-// Apps returns the app to deploy, by prefering .aab files.
-func (c Configs) Apps() []string {
-	if len(c.ApkPath) > 0 {
-		return strings.Split(c.ApkPath, "|")
-	}
+func parseAPKList(list string) []string {
+	return strings.Split(list, "|")
+}
 
-	apps := strings.Split(c.AppPath, "\n")
+func parseAppList(list string) (apps []string) {
+	for _, app := range strings.Split(list, "\n") {
+		apps = append(apps, strings.TrimSpace(app))
+	}
 	if len(apps) == 0 {
-		apps = strings.Split(c.AppPath, "|")
+		for _, app := range strings.Split(list, "|") {
+			apps = append(apps, strings.TrimSpace(app))
+		}
 	}
+	return
+}
 
-	var apks, aabs []string
-	for _, pth := range apps {
+// appPaths returns the app to deploy, by prefering .aab files.
+func (c Configs) appPaths() ([]string, []string) {
+	if len(c.ApkPath) > 0 {
+		return parseAPKList(c.ApkPath), []string{"step input 'APK file path' (apk_path) is deprecated and will be removed on 20. August, use 'APK or App Bundle file path' (app_path) instead!"}
```

Comment:
please use one of the listed date formats: https://www.englishclub.com/vocabulary/time-date.htm